### PR TITLE
Fixed docker links with updated ones

### PR DESCRIPTION
**Pull request by budhrg**
Updated with docker links in `tools/docker` directory.

### CODE REVIEW

File: tools/docker/docker-installation.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ You can either [set up sudo](http://www.projectatomic.io/blog/2015/08/why-we-don
 
 Or you can create a Unix group called `docker` and add users to it. When the Docker daemon starts, it makes the ownership of the Unix socket read/writable by the `docker` group.
 
-**Warning:** The `docker` group is equivalent to the `root` user; For details on how this impacts security in your system, see [Docker Daemon Attack Surface](https://docs.Docker.com/articles/security/#Docker-daemon-attack-surface) for details.
```
Comment (pvalena) [inline tools/docker/docker-installation.md]: Capital `D` here :)

Comment (budhrg) [inline tools/docker/docker-installation.md]: @pvalena Sorry didn't get you

**Comment by pvalena** [comment]
LGTM.

Thanks!

File: tools/docker/docker-installation.md (hunk 1 above)
Comment (pvalena) [inline tools/docker/docker-installation.md]: It's just funny we had a `docs.Docker.com` link.

Comment (budhrg) [inline tools/docker/docker-installation.md]: :) ya @pvalena , but it redirect to docs.docker.com  :)

**Pull request merged**